lang/de.yml

```
@@ -41,27 +41,27 @@ HELP_EXCLUDE_2: "Beispiele: [e:stone], [e:Notch], [e:stone,Notch]"
 HELP_HEADER: "{0} Hilfe"
 HELP_INCLUDE_1: "Fügen Sie Blöcke/Entitäten ein."
 HELP_INCLUDE_2: "Beispiele: [i:stone], [i:zombie], [i:stone,wood,bedrock]"
-HELP_INSPECT_1: "Wenn der Inspektor aktiviert ist, können Sie:"
-HELP_INSPECT_2: "Klicken Sie mit der linken Maustaste, um die Platzierung anzuzeigen."
-HELP_INSPECT_3: "Klicken Sie mit der rechten Maustaste, um benachbarte Entfernungen anzuzeigen."
-HELP_INSPECT_4: "Platzieren Sie einen Block, um die Entfernung zu sehen."
-HELP_INSPECT_5: "Legen Sie einen Block in Flüssigkeit, um die Platzierung zu sehen."
-HELP_INSPECT_6: "Klicken Sie mit der rechten Maustaste, um Truhentransaktionen anzuzeigen."
-HELP_INSPECT_7: "Tipp: Sie können einfach \"/co i\" verwenden."
+HELP_INSPECT_1: "Mit dem Inspector aktiviert, kannst du Folgendes tun:"
+HELP_INSPECT_2: "Linksklick auf einen Block, um zu sehen, wer diesen Block platziert hat."
+HELP_INSPECT_3: "Rechtsklick auf einen Block, um zu sehen, welcher benachbarte Block zerstört wurde."
+HELP_INSPECT_4: "Einen Block platzieren, um zu sehen, welcher Block an dieser Stelle zerstört wurde."
+HELP_INSPECT_5: "Einen Block in Flüssigkeit (etc.) platzieren, um zu sehen, wer sie platziert hat."
+HELP_INSPECT_6: "Rechtsklick auf eine Tür, Truhe, etc., um zu sehen, wer sie zuletzt verwendet hat."
+HELP_INSPECT_7: "Tipp: Du kannst einfach \"/co i\" verwenden, um schnelleren Zugriff zu erhalten."
```

Shortened the translation & made sure it was short enough to fit on one line in-game. I had to make the fifth line reference the fourth one because it's already very close to maximum length and there's no way to make this work in German, lol. Furthermore, I removed the periods at the end of each line because I couldn't make HELP_INSPECT_3 work with them, but I think this should be fine.

```suggestion
HELP_INSPECT_1: "Mit dem Inspector kannst du Folgendes tun:"
HELP_INSPECT_2: "Linksklicke einen Block, um zu sehen, wer ihn platziert hat"
HELP_INSPECT_3: "Rechtsklick auf einen Block zeigt Änderungen an Nachbarn an"
HELP_INSPECT_4: "Platziere einen Block, um Änderungen an seinem Ort zu sehen"
HELP_INSPECT_5: "So kann man auch Änderungen an Flüssigkeiten etc. sehen"
HELP_INSPECT_6: "Rechtsklicke Türen, Truhen, etc., um Zugriffe zu sehen"
HELP_INSPECT_7: "Tipp: Benutze \"/co i\" für schnelleren Zugriff"
```